### ISSUE TITLE
nix-direnv is always using cache/unknown

### ISSUE THREAD

**Issue by gilligan** [nix-community/nix-direnv#65]
I only ever see `.direnv/cache-unknown` and looking at the source i don't quite understand how it could be any different:

```
$ nix-instantiate --find-file nixpkgs
/nix/store/plaxbbv04m996smsi6gaqgx2lvb2yz0j-356pa04sqir8ycxni9pd0c8j9qv2z4v0-source
```

There is neither a `version-suffix` file (where would that come from? nixpkgs only has a `.version` file doesn't it?) and there is no `.git` directory because it's not a working copy.

Terribly sorry if i am somehow completely missing the point in some regard but i don't quite understand what's going on or if I am perhaps just "holding it wrong" :)

**Comment by Mic92** [comment]
This might happen if you get your channel from github directly.

**Comment by Mic92** [comment]
Better? https://github.com/nix-community/nix-direnv/pull/66

**Comment by Mic92** [comment]
fixed in #66

**Comment by gilligan** [comment]
@Mic92 thanks!
